packages/hyperverse-polygon-randompick/contracts/RandomPick.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import "@chainlink/contracts/src/v0.8/VRFConsumerBase.sol";
+
+/**
+ * Request testnet LINK and ETH here: https://faucets.chain.link/
+ * Find information on LINK Token Contracts and get the latest ETH and LINK faucets here: https://docs.chain.link/docs/link-token-contracts/
+ */
+contract RandomPick is VRFConsumerBase {
+    bytes32 internal keyHash;
+    uint256 internal fee;
+    // Where you can access your results
+    // requestId => result
+    mapping(bytes32 => int256) public results;
+    /* Helpers along the way */
+    // requestId => input
+    mapping(bytes32 => int256[]) public randomList;
+    // requestId => tenant address
+    mapping(bytes32 => address) public requestIdToAddress;
+    event StartedRandomPick(address tenant, bytes32 requestId);
+
+    /**
+     * Constructor inherits VRFConsumerBase
+     * All of this info is gotten from: https://docs.chain.link/docs/vrf-contracts/
+     *
+     * Network: Rinkeby
+     * Chainlink VRF Coordinator address: 0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B
+     * LINK token address:                0x01BE23585060835E02B77ef475b0Cc51aA1e0709
+     * Key Hash: 0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311
+     * Fee: 0.1 LINK
+     */
+    constructor()
+        VRFConsumerBase(
+            0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B, // VRF Coordinator
```

Comment:
I believe these are the contracts for Ethereum. There is some documentation around this here

https://blog.chain.link/how-to-get-a-random-number-on-polygon/

Potential contracts
```
constructor() 
        VRFConsumerBase(
            0x8C7382F9D8f56b33781fE506E897a4F1e2d17255
            0x326C977E6efc84E512bB9C30f76E30c160eD06FB
        ) public
    {
        keyHash = 0x6e75b569a01ef56d18cab6a8e71e6600d6ce853834d4a5748b720d06f878b3a4;
        fee = 100000000000000; // 0.0001 LINK
    }
```

Reply:
@jsheely cool I'll make the adjustments and push when I've got time. I'm still learning how this all fits together.

Reply:
@jsheely i think this is the blocker for the moment on this PR. I'll update my branch for testnet and then maybe we can create another issue for discussion on how to move forward on mainnet?
![Screen Shot 2022-03-21 at 7 58 20 PM](https://user-images.githubusercontent.com/83088883/159399039-b737fae6-2dc1-4367-a70f-b56830d78740.png)

Reply:
@jsheely do I need to generate the ABI to go along with updating the contract?